packages/test-snaps/src/components/Connect.tsx

```diff
@@ -62,9 +62,7 @@ export const Connect: FunctionComponent<ConnectProps> = ({
         {isLoading ? (
           <ButtonSpinner>Connecting</ButtonSpinner>
         ) : (
-          <span>
-            {isInstalled ? 'Reconnect' : 'Connect'} to {name}
-          </span>
```

Comment:
this nested <span> inside a <button> interferes when we try to interact with the button element